dotcom-rendering/src/server/prod-server.ts

```diff
@@ -102,6 +104,21 @@ export const prodServer = (): void => {
 		},
 	);
 
+	app.get(
+		'/FrontJSON',
```

Comment:
How would you get to this on prod?

Reply:
This will allow you to get the enhanced endpoint when running things like `make run-ci` which run off of the prod server.

If we find this useful and want to add it to 'prod' then we could add the endpoint in Frontend when we query with something like `www.theguardian.com/uk.json?dcr&enhanced` for example - but I would first want to discuss whether are comfortable exposing that data publicly

Reply:
> then we could add the endpoint in Frontend

I can't think of a reason - I just forgot `prod` is just config rather than where it i hosted so has more applications.